terraform/aws/analytical-platform-development/cluster/iam-policies.tf

```diff
@@ -76,6 +76,7 @@ resource "aws_iam_policy" "cluster_autoscaler" {
 ##################################################
 
 data "aws_iam_policy_document" "efs_csi_driver" {
+  #checkov:skip=CKV_AWS_356: skip requires access to multiple resources
```

Comment:
As we have only one EFS instance, we could scope this down

```
resources = [aws_efs_file_system.eks_user_homes.arn]
```

Reply:
change implemented

Reply:
fix as suggested implemented